src/Main.java

```diff
@@ -1,9 +1,37 @@
+import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) {
 
-        // Hier mag je je code schrijven voor de hoofd-opdracht
+        Integer[] numeric = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
+        String[] alphabetic = {"een", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht", "negen", "nul"};
 
+        Translator t1 = new Translator(numeric, alphabetic);
```

Comment:
Netjes gedaan, bij de uitwerkingen staan numeric en alphabetic andersom. Persoonlijk vond ik dat verwarrend en is deze volgorde logischer!